esque/cli/commands.py

```diff
@@ -428,6 +505,24 @@ def consume(
     preserve_order: bool,
     write_to_stdout: bool,
 ):
+    """Consume messages from a topic.
+
+    Read messages from a given topic in a given context. These messages can either be written
+    to files in an automatically generated directory (default behavior), or to STDOUT.
+
+    \b
+    EXAMPLES:
+    #Consume the first 10 messages from TOPIC in the current context and print them to STDOUT in order.
+    esque consume --first -n 10 --preserve-order --stdout TOPIC
+
+    \b
+    #Consume <n> messages, starting from the 10th, from TOPIC in the <source_ctx> context and write them to files.
+    esque consume --match "message.offset > 9" -n <n> TOPIC -f <source_ctx>
+
+    \b
+    #Copy source_topic in source_ctx context to destination_topic in destination_ctx.
+    esque consume -f source_ctx --stdout source_topic | esque produce -t destination_ctx --stdin destination_topic
```

Comment:
Have we actually tried that? I'm afraid the calls to `config.switch_context` might interefere with each other since they change the actual file and we have a few `Config()` invocations throughout the code which always load the context which was last set in the file. So either pass the config around or make it possible to switch the context without writing the switch back to the file.

Reply:
It more than doesn't work, it can even break esque :D